src/Apps/__stories__/OrderApp.story.tsx

```diff
@@ -1,10 +1,18 @@
 import React from "react"
+import { StorybooksRouter } from "Router/StorybooksRouter"
 import { storiesOf } from "storybook/storiesOf"
-import { OrderApp } from "../Order/OrderApp"
-import { ShippingRoute } from "../Order/Routes/Shipping"
+import { routes as orderRoutes } from "../Order/routes"
 
 storiesOf("Apps/Order Page", module).add("Shipping", () => (
-  <OrderApp me={null} params={null}>
-    <ShippingRoute order={null} />
-  </OrderApp>
+  <StorybooksRouter
+    routes={orderRoutes}
+    initialRoute="/order2/123/shipping"
+    mockResolvers={{
```

Comment:
In a follow up PR I think the API for this can be cleaned up to just 

```jsx
<StorybooksRouter
  routes={ ... }
  route={ ... } 
  resolvers={ ... } 
  state={ ... } 
/>
```

Right now it seems like the `initial` and `mock` prefixes add unnecessary noise.